client/components/NewMeetup.js

```diff
@@ -2,20 +2,164 @@ import React, {Component} from 'react';
 import { connect } from 'react-redux';
 import {  } from '../store';
 import { Redirect } from 'react-router-dom';
+import * as _ from 'lodash';
+import axios from 'axios';
+import {daysInMonth} from '../../utils';
 
 class NewMeetup extends Component {
-    constructor(){
+    constructor(props){
         super();
+        let { user } = props,
+            now = new Date(),
+            now_year = now.getFullYear(),
+            now_month = now.getMonth() + 1,
+            now_date = now.getDate(),
+            now_hour = now.getHours();
+
+
+        this.state = {
+          avail_years: [2017, 2018],
+          avail_months: _.range(1, 13),
+          avail_dates: _.range(1, daysInMonth(now_year, now_month) + 1),
+          avail_hours: _.range(24),
+          input_year:  -1,
+          input_month: -1,
+          input_date: -1,
+          input_hour: -1,
+          input_friend: -1,
+          input_err: '',
+          input_success: ''
+        };
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleSubmit(event) {
+      event.preventDefault();
+      let {input_year, input_month, input_date, input_hour, input_friend} = this.state;
+      const {user} = this.props;
+      // console.log(this.state);
+      axios.post(`/api/meetup/add/${user.id}`, {
```

Comment:
Do you want to do the axios call here or would it be better to do it in a thunk?

Reply:
I was debating about that. But those doesn't change any redux state, so no dispatch. Feel like an overkill to create all the reducer files just to add a thunk that doesn't have anything to do with redux state.